backstop/tests/color.html

```diff
@@ -0,0 +1,255 @@
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
+          <li class="demo-text-primary">Primary</li>
+          <li class="demo-text-positive">Positive</li>
+          <li class="demo-text-warning">Warning</li>
+          <li class="demo-text-negative">Negative</li>
+        </ul>
+      </div>
+      <div class="demo-background-2">
+        <h2 class="iui-text-title iui-text-spacing">Background 2</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
+          <li class="demo-text-primary">Primary</li>
+          <li class="demo-text-positive">Positive</li>
+          <li class="demo-text-warning">Warning</li>
+          <li class="demo-text-negative">Negative</li>
+        </ul>
+      </div>
+      <div class="demo-background-3">
+        <h2 class="iui-text-title iui-text-spacing">Background 3</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
+          <li class="demo-text-primary">Primary</li>
+          <li class="demo-text-positive">Positive</li>
+          <li class="demo-text-warning">Warning</li>
+          <li class="demo-text-negative">Negative</li>
+        </ul>
+      </div>
+      <div class="demo-background-4">
+        <h2 class="iui-text-title iui-text-spacing">Background 4</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
+        </ul>
+      </div>
+      <div class="demo-background-5">
+        <h2 class="iui-text-title iui-text-spacing">Background 5</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
+        </ul>
+      </div>
+
+      <div class="demo-status-primary">
+        <h2 class="iui-text-title iui-text-spacing">Background primary</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+        </ul>
+      </div>
+      <div class="demo-status-positive">
+        <h2 class="iui-text-title iui-text-spacing">Background positive</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+        </ul>
+      </div>
+      <div class="demo-status-warning">
+        <h2 class="iui-text-title iui-text-spacing">Background warning</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+        </ul>
+      </div>
+      <div class="demo-status-negative">
+        <h2 class="iui-text-title iui-text-spacing">Background negative</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+        </ul>
+      </div>
+
+      <div class="demo-status-overlay-primary">
+        <h2 class="iui-text-title iui-text-spacing">Overlay primary</h2>
+        <ul>
+          <li class="iui-text-block">Text</li>
+          <li class="iui-text-block iui-text-muted">Muted</li>
```

Comment:
I expanded on the tests on this page and discovered yesterday that muted text on overlay primary is failing in light HC.  This is the only fail I am finding across both light & dark HC themes.  Solutions:
1. Increase `opacity-3` to passing.
2. Change muted text to use `opacity-2`.
3. Remove instances of muted text over primary overlay.

Muted text over primary overlay are present in header buttons, hovered select menu items for examples.

Reply:
This is addressed in #529.